conjur/init/conjurrc_data.py

```diff
@@ -16,6 +29,17 @@ def __init__(self, appliance_url = None, account = None, cert_file = None):
         self.cert_file = cert_file
         self.plugins = []
 
+    @classmethod
+    def load_conjurrc(cls):
+        """
+        Method that loads the conjurrc into an object
+        """
+        if os.path.exists(conjur.constants.DEFAULT_CONFIG_FILE):
+            with open(conjur.constants.DEFAULT_CONFIG_FILE, 'r') as conjurrc:
+                return load(conjurrc, Loader=Loader)
```

Comment:
Loader=Loader is redundant.
```suggestion
                return load(conjurrc, Loader)
```

Reply:
I don't think it is because default Loader is None